src/Modules/remove.js

```diff
@@ -0,0 +1,24 @@
+import { save } from "./save";
+
+export function del(list) {
+  const itemList = document.querySelector('.list-items');
+  itemList.addEventListener('click', function (e) {
+    if (e.target.classList.contains('delete')) {
+      const listItem = e.target.closest('li');
+      if (listItem && itemList.contains(listItem)) {
+        itemList.removeChild(listItem);
+        const id = parseInt(e.target.getAttribute('data-id'));
+        const index = list.findIndex(item => item.index === id);
+        if (index !== -1) {
+          list.splice(index, 1);
+          for (let i = index; i < list.length; i++) {
+            list[i].index = i;
+          }
+          save(list);
+        }
+      }
+    }
```

Comment:
- [ ] Great job on implementing the update tasks! One suggestion for improvement is to consider starting the index at 1 instead of 0. Additionally, it's commendable how you've successfully marked the changes in the local storage by updating the values to true when a task is marked and false when it's not. Keep up the good work! This was requested in [the previous review](https://github.com/As1imwe-Mark/To_Do_List-App/pull/2#discussion_r1264151422) but not yet addressed , KIndly fix this :point_down: 
![Screenshot from 2023-07-15 04-27-38](https://github.com/As1imwe-Mark/To_Do_List-App/assets/88894525/fd686761-0185-4c23-ab56-8b4a6c1f5298)